jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -73,6 +74,53 @@ local excludedRules = [
       { alert: 'PrometheusErrorSendingAlertsToAnyAlertmanager' },
     ],
   },
+  {
+    name: 'kube-prometheus-node-recording.rules',
+    rules: [
+      { alert: 'instance:node_cpu:rate:sum' },
+      { alert: 'instance:node_network_receive_bytes:rate:sum' },
+      { alert: 'instance:node_cpu:ratio' },
+      { alert: 'instance:node_network_transmit_bytes:rate:sum' },
+    ],
+  },
+  {
+    name: 'node.rules',
+    rules: [
+      { record: 'node:node_num_cpu:sum' },
+    ],
+  },
+  {
+    name: 'openshift-build.rules',
+    rules: [
+      { record: 'build_error_rate' },
```

Comment:
we remove the rule at the source:

https://github.com/openshift/cluster-monitoring-operator/blob/6e4865360447e30b326571364a19fb231c15b75a/jsonnet/rules.libsonnet#L509-L521